onboarding/engineering-introduction.md

```diff
@@ -1,46 +1,28 @@
-Awesome.
+### Work hours
+Depends on the person/where they are located! We try to schedule meetings at times that are amenable to people being remote (from Amsterdam to Seattle on the engineering team), so they'll usually happen in the middle of the day EST.
 
-Some team members have backgrounds in art, but of course not all.
-[/recommendations/art.md](/recommendations/art.md) has some useful links if you want to read more about the art
-world, or keep up with the latest news.
+Ask your mentor/manager what their general work hours are. Artsy has no policy around this, so use your judgment and be open with your team about when you expect to be available.
+
+### Vacation
+Artsy has an open vacation policy. This doesn't mean "don't take vacation." _Please take time off._ Most people take off for bank holidays.
+
+Join the [Design/Product OOO](https://calendar.google.com/calendar/embed?src=artsymail.com_gl81jptn59gjfv1kg0fer1i4jo%40group.calendar.google.com&ctz=America%2FNew_York) calendar and use that to set when you know you'll be out of the office (you don't generally need to post remote work here unless it's out of the ordinary and would benefit from wider communication i.e. you'll be working for a month from somewhere else).
 
 ### Chat
 
 Most of our day-to-day business is conducted in Slack. Teams, offices, and common concerns frequently have their
 own channels. For more information on our usage of Slack, see [`/culture/slack.md`](/culture/slack.md).
 
-### GitHub
-
-Artsy stores source code on [GitHub](https://github.com/artsy). Make sure you have an account and are part of the
-Artsy organization. If you can't visit [this page](https://github.com/artsy/gravity), then you don’t have the right
-access. Your mentor can get you sorted out.
+### Events
 
-Our projects use physics terms as code names, starting with [Gravity](https://github.com/artsy/gravity) (inspired
-by [Zachary Coffin's "Temple of Gravity"](http://www.zacharycoffin.com/work/temple-of-gravity)).
-
-See the [engineering projects map](https://github.com/artsy/potential/wiki/Project-List) for a comprehensive list
-of our (many) repos and who owns them. It can be a bit overwhelming, so here are some important ones:
-
-- [Gravity](https://github.com/artsy/gravity): Artsy's main API
-- [Force](https://github.com/artsy/force): Artsy web site ([www.artsy.net](https://www.artsy.net))
-- [Eigen](https://github.com/artsy/eigen): iOS app
-- [Volt](https://github.com/artsy/volt): Content Management system for partners
-
-### AWS
-
-Many applications depend on Amazon Web Services. Your mentor or anyone else in Engineering can set you up with an
-IAM profile within our organization's account (see
-[instructions](https://github.com/artsy/potential/wiki/Platform-FAQ#add-a-new-aws-user)).
-
-### Engineering-wide Standup
+See the [events list](/events) for a list of our recurring, engineering-wide events.
```

Comment:
This strikes me as something that could be tedious to maintain- could we link directly to the calendar itself, a calendar page in artsy atlas or some other landing page for non-OSS stuff?

Reply:
cc @ds300 - thoughts?

Reply:
We can definitely link to our team calendar - here's the link: https://calendar.google.com/calendar?cid=YXJ0c3ltYWlsLmNvbV9nODFpbzRhOThkZHZuMWloMWEzbG0yb2NkNEBncm91cC5jYWxlbmRhci5nb29nbGUuY29t